app/build.gradle

```diff
@@ -50,6 +50,10 @@ android {
             dimension "version"
         }
     }
+
+    packagingOptions {
+        exclude 'thirdpartynotice.txt'
+    }
```

Comment:
Why do we need this?

Reply:
Because Azure IOT Sdk doesn't work if we don't exclude this file